aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -206,10 +206,11 @@ static inline void linearSolveCheckInputs(const Tensor& self, const Tensor& A, c
 }
 
 // Validates input shapes for operations on batches of square matrices (inverse, cholesky, symeig, eig)
-static inline void squareCheckInputs(const Tensor& self) {
-  TORCH_CHECK(self.dim() >= 2, "Tensor of matrices must have at least 2 dimensions. ");
+static inline void squareCheckInputs(const Tensor& self, const char* const f_name) {
+  TORCH_CHECK(self.dim() >= 2, f_name, ": The input tensor must have at least 2 dimensions.");
```

Comment:
nit: `f_name, "(): ..."`. Looks like this is a generic style convention for functions.

Reply:
Leaving as it is for now, as this is the format the other errors in linalg and advanced indexing follow.